inspirehep/records/marshmallow/literature/common/author.py

```diff
@@ -43,6 +45,11 @@ def get_last_name(self, data):
 
         return missing
 
+    def get_recid(self, data):
```

Comment:
this didn't exist anywhere else before?

Reply:
AFAIK nope

Reply:
ok I guess we can leave it like that, until we need it elsewhere.